### DISCUSSION TITLE
Disable code splitting in a child compiler

### DISCUSSION THREAD

**Discussion by tavoyne** [webpack/webpack#15399] [Q&A] [answered]
Is there any way to make Webpack interpret dynamic imports as a regular ones in a child compiler?

I'm building a [plugin for doing SSG](https://github.com/tavoyne/ssg-webpack-plugin/tree/master/plugin) and I don't want the dynamic imports encountered by the child compiler to cause the emission of additional chunks. I'd like to have one self-contained output asset per entry point.

**Comment by vankop** [comment, marked as answer]
https://webpack.js.org/plugins/limit-chunk-count-plugin/

**Reply by tavoyne** [reply]
Thanks!
